backend/app/adapter/db/user_test.go

```diff
@@ -99,7 +99,8 @@ func TestUserSql_GetByEmail(t *testing.T) {
 			mdtest.Equal(t, nil, err)
 			defer db.Close()
 
-			expQuery := fmt.Sprintf(`^SELECT ".+",".+",".+",".+",".+" FROM "%s" WHERE "%s"=.+$`, table.User.TableName, table.User.ColumnEmail)
+			expQuery := fmt.Sprintf(`^SELECT ".+",".+",".+",".+",".+" FROM "%s" WHERE "%s"=.+$`,
```

Comment:
Please remember to move this also.

Reply:
This is just part of reformatting not related to search